flexmeasures/cli/data_add.py

```diff
@@ -176,7 +190,46 @@ def new_account(name: str, roles: str, consultancy_account: Account | None):
     if account is not None:
         click.secho(f"Account '{name}' already exists.", **MsgStyle.ERROR)
         raise click.Abort()
-    account = Account(name=name, consultancy_account=consultancy_account)
+
+    # Validate color format ---------------------------------------------------
+    # Regular expression for valid hex color code
+    hex_color_pattern = re.compile(r"^#?([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
```

Comment:
I left it here as this is the only place it's being used, what do you think

Reply:
Also, i have the validation written inside the command function itself since its small, but I will also like to know if using click commands is a standard among the team so I can stick to that for validation from now on.

Reply:
My motivations for moving into a click validation:

- cleaner code in the command function (this one): ideally, only the business logic is in there so it's easy to grasp what it does
- click gives nicer error messages when it validates parameters than when we do this in the function

And yes, this is somewhat of a team practice :)

Reply:
Ok i see, thanks for clarifying i will implement the click validation on this